server/index.js

```diff
@@ -76,6 +75,27 @@ app.get('/api/garage/recent-history/:vehicleId', (req, res, next) => {
     })
     .catch(err => next(err));
 });
+
+app.post('/api/garage/add-record/:vehicleId', (req, res, next) => {
+  const { vehicleId } = req.params;
+  const { record, date, mileage, cost } = req.body;
+  if (vehicleId < 1 || !Number(vehicleId)) {
+    throw new ClientError(400, 'vehicleId must be a positive integer');
+  }
+  if (!record || !date || !mileage || !cost) {
+    throw new ClientError(400, 'Maintenance name, date, mileage, and cost are required');
+  }
+  const sql = `
+    insert  into "records" ("vehicleId", "maintenanceName", "datePerformed", "mileage", "cost")
+    values  ($1, $2, $3, $4, $5)
```

Comment:
Include a `returning` clause to get back the inserted row.

Reply:
added returning clause to sql query
![image](https://user-images.githubusercontent.com/48267398/163653340-3048fdf5-e403-4e18-af2e-0de853ccc99f.png)